### PR TITLE
Update Swift 2.3 Branch Readme Documentation

### PR DESCRIPTION
**Pull request by Twigz**
Updates the `Readme.md` documentation on the `swift-2.3` branch to reflect appropriate steps for installation.

### CODE REVIEW

**Comment by mliberatore** [comment]
> > **Note:** Branches for Swift 2.3 ([`swift-2.3`](https://github.com/Lickability/PinpointKit/tree/swift-2.3)) and 3.0 ([`swift-3.0`](https://github.com/Lickability/PinpointKit/tree/swift-3.0)) are being kept up-to-date as Xcode 8 betas are released.

Can this line be updated to just be another requirement, i.e.

- Swift 2.3

**Comment by mliberatore** [comment]
> Xcode 7.3+

Should now be

> Xcode 8.0+

**Comment by mliberatore** [comment]
Note we’ll still have to address #128 on `swift-2.3` before freezing the branch from new features, and optionally #165. This should be separate from this PR, though.

**Comment by mliberatore** [comment]
All links appear :+1:
All sample code worked :+1:
CocoaPods install instructions worked :+1:

With CocoaPods, I see the following after integrating PinpointKit into a project and opening the workspace. Is this possible to avoid?

screen shot 2016-10-14 at 4 09 16 pm

Moving onto testing Carthage...

File: README.md [hunk 1]
```diff
@@ -82,7 +84,7 @@ $ brew install carthage
 To integrate PinpointKit into your Xcode project using Carthage, specify it in your `Cartfile`:
 
 ```ogdl
-github "Lickability/PinpointKit" ~> 0.9
+github "Lickability/PinpointKit" ~> "swift-2.3"
```
Comment (mliberatore) [inline README.md]: I get `Parse error: unexpected trailing characters in line: github "Lickability/PinpointKit" ~> "swift-2.3"`

I think this needs to be `github "Lickability/PinpointKit" "swift-2.3"`

**Comment by mliberatore** [comment]
> Drag the built `PinpointKit.framework` from the Carthage build folder into the “Embedded Binaries” section.

For Carthage, we’ll need to add a step to this process, or at least a note somewhere in here to ensure the following, from the same interface:

- In the tab bar at the top of that window, open the “Build Settings” panel.
- Ensure that “Use Legacy Swift Version” is set to “Yes”


Otherwise, we end up with this error:

screen shot 2016-10-14 at 4 30 55 pm

**Comment by mliberatore** [comment]
> For Carthage, we’ll need to add a step to this process...

Same for Manual install. Other than that, the steps there are working 👌

**Comment by Twigz** [comment]
@mliberatore You know what would be nice. Like a "comments addressed" button.

**Comment by mliberatore** [comment]
Looks good :+1:, any idea about the [alert with CocoaPods](https://github.com/Lickability/PinpointKit/pull/179#issuecomment-253909900)?

**Pull request merged**